src/main/java/profiler/generation/R1CSConstruction.java

```diff
@@ -503,14 +550,8 @@ void printMatrix(ArrayList<Tuple2<Long, FieldT>> data,
     // B: n2 by n3
     // C: n1 by n3
     // Z are the partial products, and S are the partial sums
-    public static <FieldT extends AbstractFieldElementExpanded<FieldT>> Tuple3<R1CSRelationRDD<FieldT>,
-            Assignment<FieldT>,
-            JavaPairRDD<Long, FieldT>>
-    matmulConstruct(final int n1,
-                    final int n2,
-                    final int n3,
-                    final FieldT fieldFactory,
-                    final Configuration config) {
+    public static <FieldT extends AbstractFieldElementExpanded<FieldT>> Tuple3<R1CSRelationRDD<FieldT>, Assignment<FieldT>, JavaPairRDD<Long, FieldT>>
```

Comment:
For merging back into main DIZK it's not great to have the IDE change format for code that we didn't actually touch. Does it do that automatically on save? Can we only have auto-format turned on for code we touched?

Reply:
That was me. I can try to refrain form doing this. Good thinking.